gateway-service/src/chef/chef.service.ts

```
@@ -0,0 +1,78 @@
+      throw error;
+    }
+  }
+
+  async postLoginChef(body: ChefLoginModel[]): Promise<any> {
+    try {
+      const result = await lastValueFrom(
+        this.chefClient.send('chef/login', body),
+      );
+      return result;
+    } catch (error) {
+      throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)